containers/daap-update-schema/src/var/task/update_schema.py

```diff
@@ -56,6 +60,18 @@ def handler(event, context):
             message="Update not allowed",
         )
 
-    return format_response_json(
-        status_code=HTTPStatus.OK, body={"version": new_version, "changes": changes}
-    )
+    # check if major or minor
+    if not int(new_version.split(".")[0][-1]) == int(schema.version.split(".")[0][-1]):
+        body_dict = {
+            "version": new_version,
+            "increment_type": "major",
+            "changes": changes,
+            **copy_response,
```

Comment:
Should we change `VersionManager.update_schema` to return this dictionary, given all of this information is handled as part of the `update_schema` function anyway?

Reply:
Yeah possibly. My thoughts on it were that it should be done as a wider review/refactor of all responses. I think we'll need to ensure better consistency as we've been doing our own and not following a strict framework (i haven't at least...), other than to be informative/useful responses.